docs/user/admin/settings.rst

```diff
@@ -305,7 +305,7 @@ SQL query::
     {
       "error": {
         "reason": "Invalid SQL query",
-        "details": "DELETE clause is disabled by default and will be deprecated. Using the plugins.sql.delete.enabled setting to enable it",
+        "details": "Failed to parse query due to offending symbol [DELETE] at: 'DELETE' <--- HERE... More details: Expecting tokens in {<EOF>, 'DESCRIBE', 'SELECT', 'SHOW', ';'}",
```

Comment:
I think the originl message is much clear

Reply:
https://github.com/opensearch-project/sql/pull/839#discussion_r981653387

Reply:
Revered in b5ad3c9.